typescript/src/deposit-sweep.ts

```diff
@@ -36,7 +36,7 @@ import { createTransactionProof } from "./proof"
  *          - the sweep transaction hash,
  *          - the new wallet's main UTXO produced by this transaction.
  */
-export async function sweepDeposits(
+export async function submitDepositSweepTransaction(
```

Comment:
Non-blocking:

Similar to above comments. We could change the function description to:

"**Submits a sweep transaction** by combining all the provided **P2(W)SH** UTXOs and broadcasting a Bitcoin P2WPKH deposit sweep transaction."

Reply:
See https://github.com/keep-network/tbtc-v2/pull/364/commits/e3da6c934d74540d8265dcbd89bbd717385e63cd